### DISCUSSION TITLE
Different color for debug

### DISCUSSION THREAD

**Discussion by PHLAK** [opcodesio/log-viewer#287] [Ideas]
Can we get a different color for each log level? Specifically `info` and `debug` levels are the same and this makes it harder to distinguish the difference at a glance.

<img src="https://github.com/opcodesio/log-viewer/assets/53531/ac5f9dcc-54ce-49c2-9d89-1f8aa29e6ddc" width="50%">

It would be much easier to tell the difference if they were given distinct colors.

<img src="https://github.com/opcodesio/log-viewer/assets/53531/15e240c1-b89a-49c8-90fc-0c5df916950e" width="50%">

The same applies to both `critical` and `error`.

**Comment by PHLAK** [comment]
To expand on this, I think of increasing log severity levels mapping to colors on the rainbow (e.g. ROYGBIV) where more severe logs tend towards red and more informational logs tend towards violet. For example.
  
  - **Debug:** Purple (Violet)
  - **Info:** Blue
  - **Warning:** Yellow
  - **Error:** Orange
  - **Critical:** Red
 
Though I can see reason for both error and critical being red.

**Reply by arukompas** [reply]
That is an interesting suggestion. I think of colours as "how important it is for me to take action on this", which is what each severity represents as well. 

If there's too many colours, it might start to get confusing though.

Is there ever a case where seeing both Debug and Info logs in blue makes it confusing?

**Reply by PHLAK** [reply]
We use informational logs to track a customer through our app (e.g. ordering, paying, status of their order as it progresses, etc.) and we use debug to track code execution (e.g. event fired, running job, etc.). So when looking at the logs the color makes it easier to understand what's happening.

Is it critical that they be different colors? No, but it would helpful.

**Reply by gabbykweka** [reply]
Im also in case of needing the multiple colors. Its alway an issue if you use the logs for more than just tracking errors. My suggestion for worrying that too many color may be confusing, may be it should be configurable. The user to choose which mode of color set to use or even to set his own color option base on the status. Just idea!
